src/boss/lambdaStore.go

```diff
@@ -0,0 +1,300 @@
+package boss
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/open-lambda/open-lambda/ol/common"
+)
+
+type LambdaStore struct {
+	StorePath    string
+	Lambdas      []LambdaEntry
+	HTTPTriggers []HTTPEntry
+	CronTriggers []CronEntry
+}
+
+type LambdaEntry struct {
```

Comment:
Do we need all this?  It feels redundant with your code in common: https://github.com/open-lambda/open-lambda/blob/a8ad2825428b10aaed7d8d6472eae00ac91fbc10/src/common/lambdaConfig.go#L13